src/common.c

```diff
@@ -1129,7 +1140,17 @@ ssize_t ofi_bsock_sendv(struct ofi_bsock *bsock, const struct iovec *iov,
 	msg.msg_iov = (struct iovec *) iov;
 	msg.msg_iovlen = cnt;
 
-	ret = ofi_sendmsg_tcp(bsock->sock, &msg, MSG_NOSIGNAL);
+	if (bytes > bsock->zerocopy_size) {
+		ret = ofi_sendmsg_tcp(bsock->sock, &msg,
+				      MSG_NOSIGNAL | OFI_ZEROCOPY);
+		if (ret >= 0) {
+			bsock->async_index++;
+			*len = ret;
```

Comment:
Same question as before:  shouldn't we change the API of ofi_bsock_sendv() and always return the number of bytes transferred (or waiting for transmission) in the in/out argument `len`, even if zerocopy is not used?